src/dockleHelper.ts

```diff
@@ -87,6 +87,23 @@ export function getText(dockleStatus: number): string {
     return `**Best Practices Violations** -${clusteredViolations ? clusteredViolations : '\nNone found.'}`;
 }
 
+export function getDockleFilteredOutput(): any {
+    const dockleOutputJson = getDockleOutput();
+    let filteredVulnerabilities = [];
+    dockleOutputJson[KEY_DETAILS].forEach(ele => {

```

Comment:
don't use short forms which aren't intuitive.

Reply:
Changes to cis